packages/cli/src/docs-mcp-server/index.ts

```diff
@@ -0,0 +1,213 @@
+		{
+			url: z
+				.string()
+				.describe(
+					'URL of a specific SDK page to fetch. Format: /sdk/...'
+				)
+		},
+		async ({ url }) => {
+			const content = await fetchDocsPost(
+				`https://langbase.com/docs${url}`
+			);
+			return {
+				content: [
+					{
+						type: 'text',
+						text: content
+					}
+				]
+			};
+		}
+	);
+
+	server.tool(
+		'examples-tool',
+		'Fetches code examples and sample implementations from the documentation. Use this tool when users specifically request examples, sample code, or implementation demonstrations. This tool provides practical code snippets and complete working examples that demonstrate how to implement various features.',
+		{
+			url: z
+				.string()
+				.describe(
+					'URL of a specific examples page to fetch. Format: /examples/...'
+				)
+		},
+		async ({ url }) => {
+			const content = await fetchDocsPost(
+				`https://langbase.com/docs${url}`
+			);
+			return {
+				content: [
+					{
+						type: 'text',
+						text: content
+					}
+				]
+			};
+		}
+	);
+
+	server.tool(
+		'guide-tool',
+		'Fetches detailed guides and tutorials from the documentation. Use this tool when users explicitly request guides, tutorials, or how-to content. This tool provides step-by-step instructions and practical examples for implementing various features.',
+		{
+			url: z
+				.string()
+				.describe(
+					'URL of a specific guide page to fetch. Format: /guides/...'
+				)
+		},
+		async ({ url }) => {
+			const content = await fetchDocsPost(
+				`https://langbase.com/docs${url}`
+			);
+			return {
+				content: [
+					{
+						type: 'text',
+						text: content
+					}
+				]
+			};
+		}
+	);
+
+	server.tool(
+		'api-reference-tool',
+		'Fetches API reference documentation. Use this tool ONLY when the user explicitly asks about API endpoints, REST API calls, or programmatically creating/updating/deleting resources (like pipes, memory, etc.) through the API interface. For general SDK implementation questions, use the sdk-documentation-fetcher instead.',
+		{
+			url: z
+				.string()
+				.describe(
+					'URL of a specific api-reference page to fetch. Format: /api-reference/...'
+				)
+		},
+		async ({ url }) => {
+			const content = await fetchDocsPost(
+				`https://langbase.com/docs${url}`
+			);
+			return {
+				content: [
+					{
+						type: 'text',
+						text: content
+					}
+				]
+			};
+		}
+	);
+
+	async function main() {
+		const transport = new StdioServerTransport();
+
+		try {
+			await server.connect(transport);
+			console.error('Langbase service MCP Server running on stdio');
```

Comment:
Why are we using `console.error` here? This seems like not an error.

Also, will it be showed in MCP client? If no, maybe we can remove it?